impeller/renderer/backend/vulkan/command_encoder_vk.h

```diff
@@ -68,11 +67,14 @@ class CommandEncoderVK {
  private:
   friend class ContextVK;
 
-  std::weak_ptr<const DeviceHolder> device_holder_;
-  std::shared_ptr<QueueVK> queue_;
+  bool Prepare() const;
+
   std::shared_ptr<FenceWaiterVK> fence_waiter_;
-  std::shared_ptr<TrackedObjectsVK> tracked_objects_;
+  std::weak_ptr<const ContextVK> context_;
+  mutable std::shared_ptr<QueueVK> queue_;
```

Comment:
Remove the `const` qualifier from the affected methods instead of making these members mutable.